Lib/email/header.py

```diff
@@ -152,6 +158,23 @@ def decode_header(header):
     return collapsed
 
 
+
+def decode_header_to_string(header):
+    """Decode a message header into a string.
+
+    header may be a string that may or may not contain RFC2047 encoded words,
+    or it may be a Header object; in the latter case, this is equivalent to
+    str(header).
+
+    An email.errors.HeaderParseError may be raised when certain decoding error
```

Comment:
This comment should be lifted into the stdlib documentation.

Reply:
Ack, will do.